{
  "id": "CVE-2022-42731",
  "assigner": "cve@mitre.org",
  "Published": "2022-10-11T12:15:10.053Z",
  "Modified": "2022-10-11T17:05:57.023Z",
  "last-modified": "2022-10-11T17:05:57.023Z",
  "summary": "mfa/FIDO2.py in django-mfa2 before 2.5.1 and 2.6.x before 2.6.1 allows a replay attack that could be used to register another device for a user. The device registration challenge is not invalidated after usage.",
  "access": {
    "authentication": null,
    "complexity": null,
    "vector": null
  },
  "impact": {
    "availability": null,
    "confidentiality": null,
    "integrity": null
  },
  "cvss": null,
  "exploitabilityScore": null,
  "impactScore": null,
  "cvss-vector": null,
  "references": [
    "https://github.com/mkalioby/django-mfa2/blob/0936ea253354dd95cb127f09d0efa31324caef27/mfa/FIDO2.py#L58",
    "https://github.com/mkalioby/django-mfa2/releases/tag/v2.5.1-release",
    "https://github.com/mkalioby/django-mfa2/releases/tag/v2.6.1-release"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:django-mfa2_project:django-mfa2:-:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.7.11:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.9:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.1.0:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.1.0:beta1:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.3.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.4.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.5:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.5:beta1:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.6.0:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.6.0:rc1:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:django-mfa2_project:django-mfa2:-:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.7.11:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:1.9:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.0.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.1.0:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.1.0:beta1:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.3.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.4.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.5:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.5:beta1:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.6.0:-:*:*:*:*:*:*",
    "cpe:2.3:a:django-mfa2_project:django-mfa2:2.6.0:rc1:*:*:*:*:*:*"
  ],
  "vendors": [
    "django-mfa2_project"
  ],
  "products": [
    "django-mfa2"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:django-mfa2_project:django-mfa2"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:django-mfa2_project:django-mfa2"
  ],
  "cwe": "CWE-294",
  "vulnerable_configuration_cpe_2_2": []
}